brd-ios/breadwallet/src/Redesign/Views/MainSwapView.swift

```diff
@@ -189,44 +189,53 @@ class MainSwapView: FEView<MainSwapConfiguration, MainSwapViewModel> {
         didTapToAssetsSelection?()
     }
     
+    func toggleSwapButtonState(_ value: Bool) {
```

Comment:
maybe nitpicking, but from a 'naming' perspective
toggleSwapButtonState would indicate that the press toggles the value...
but it actually sets the state... so maybe setToggleSwapButtonState(to state: Bool)

Reply:
Was just looking at this. Updated.